### PR TITLE
Fix issue with auto generating code

### PR DESCRIPTION
**Pull request by RenukaFernando**
### Proposed changes in this pull request
- Resolve issue with generating code with older version of the plugin `swagger2msf4j-maven-plugin`
- Add the missing configurations for the `swagger2msf4j-maven-plugin`.
- Generate code.
  - org.wso2.carbon.apimgt.rest.api.admin
  - org.wso2.carbon.apimgt.rest.api.analytics
  - org.wso2.carbon.apimgt.rest.api.core
  - org.wso2.carbon.apimgt.rest.api.publisher
  - org.wso2.carbon.apimgt.rest.api.store
- Fix issue with the commit https://github.com/wso2/carbon-apimgt/commit/9767d856fa349f31062a9afcf54f97aabbcdc3c2#diff-196b3adc718374b8b751bdc5c788a1ff

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.store/src/gen/java/org/wso2/carbon/apimgt/rest/api/store/ImportApi.java [hunk 1]
```diff
@@ -52,7 +54,7 @@
         @io.swagger.annotations.Authorization(value = "OAuth2Security", scopes = {
             @io.swagger.annotations.AuthorizationScope(scope = "apim:subscribe", description = "Subscribe API")
         })
-    }, tags={ "Application (Individual)", })
+    }, tags={ "Import Configuration", })
```
[the remark this answers is not in the document]
Comment (randilu) [reply components/apimgt/org.wso2.carbon.apimgt.rest.api.store/src/gen/java/org/wso2/carbon/apimgt/rest/api/store/ImportApi.java]: noted

[the remark this answers is not in the document]
Comment (randilu) [reply components/apimgt/org.wso2.carbon.apimgt.rest.api.store/src/gen/java/org/wso2/carbon/apimgt/rest/api/store/ImportApi.java]: The tag should be "Application (Individual)"

[the remark this answers is not in the document]
Comment (RenukaFernando) [reply components/apimgt/org.wso2.carbon.apimgt.rest.api.store/src/gen/java/org/wso2/carbon/apimgt/rest/api/store/ImportApi.java]: Fixed with the commit: https://github.com/wso2/carbon-apimgt/pull/4833/commits/97c41669bb6fe7e26ed6bc1603b6065f85e6e15c

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.store/src/main/java/org/wso2/carbon/apimgt/rest/api/store/factories/ExportApiServiceFactory.java [hunk 2]
```diff
@@ -1,22 +1,3 @@
...
- *
- *  Unless required by applicable law or agreed to in writing,
- *  software distributed under the License is distributed on an
- *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- *  KIND, either express or implied.  See the License for the
- *  specific language governing permissions and limitations
- *  under the License.
- *
- */
-
```
[the remark this answers is not in the document]
Comment (RenukaFernando) [reply components/apimgt/org.wso2.carbon.apimgt.rest.api.store/src/main/java/org/wso2/carbon/apimgt/rest/api/store/factories/ExportApiServiceFactory.java]: Created an issue
https://github.com/sanjeewa-malalgoda/swagger2MSF4J/issues/17

**Pull request merged**